src/test/java/org/folio/rest/api/KafkaTopicsExistsTest.java

```diff
@@ -0,0 +1,64 @@
+package org.folio.rest.api;
```

Comment:
This doesn't look like an API test for me and could go into package org.folio.services.kafka.topic.

Reply:
I put it there as it uses StorageTestSuite / TestBase.. You still think it should go there?

Reply:
Moving it as everything is sufficiently public.